kview/src/main/java/dev/ikm/komet/kview/controls/skin/AutoCompleteTextFieldSkin.java

```diff
@@ -193,17 +207,31 @@ public AutoCompletePopupSkin(AutoCompletePopup<T> control, AutoCompleteTextField
 
             autoCompleteListView.prefWidthProperty().bind(control.prefWidthProperty());
 
-            autoCompleteListView.prefHeightProperty().bind(new DoubleBinding() {
-               {
-                   super.bind(control.getItems());
-               }
+            DoubleBinding heightBinding = new DoubleBinding() {
+                {
+                    super.bind(control.getItems(), autoCompleteTextField.maxNumberOfSuggestionsProperty());
+                }
+
+                @Override
+                protected double computeValue() {
+//                    Node cell = autoCompleteListView.lookup(".list-cell");
+//                    double cellHeight;
+//                    if (cell != null) {
+//                        System.out.println("------------- CELL != NULL");
+//                        cellHeight = cell.getBoundsInLocal().getHeight();
+//                        System.out.println("------------- CELL HEIGHT ----- " + cellHeight);
+//                    } else {
+//                        System.out.println("------------- CELL IS NULL!!!");
+//                        cellHeight = 16;
+//                    }
```

Comment:
Can this be removed? Or is this still needed for debugging?

Reply:
It's needed in case we start to want to allow custom cell factories with different sizes